src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] finalFigures = new Figure[10];
+
+        for (int i = 0; i < finalFigures.length / 2; i++) {
+            finalFigures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = finalFigures.length / 2; i < finalFigures.length; i++) {
+            finalFigures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
```suggestion
        for (int i = 0; i < finalFigures.length / 2; i++) {
            finalFigures[i] = figureSupplier.getRandomFigure();
        }

        for (int i = finalFigures.length / 2; i < finalFigures.length; i++) {
            finalFigures[i] = figureSupplier.getDefaultFigure();
        }
```

use one loop for this